onadata/apps/api/tests/viewsets/test_user_profile_viewset.py

```diff
@@ -1655,16 +1645,11 @@ def test_cache_isolation_between_users_retrieve(self):
         self.assertEqual(cached_data_b_own["first_name"], "UserB")
         self.assertEqual(cached_data_b_own["email"], user_b_data["email"])
 
-        # anonymous user retrieves user B's profile
+        # anonymous user retrieves user B's profile - should be blocked
```

Comment:
Since anonymous users are now blocked, do we need test cases for `anonymous user retrieves user B's profile - should be blocked`? Seems test `test_profiles_get_anon` is sufficient